test/stream/api/sink/create-011-exists-409/004-poll-sink/request.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do we need to poll sink if we just want to return the colision `409`? Or is it different use case?